standards/alerting.md

```diff
@@ -1,6 +1,6 @@
 # Monitoring & Alarms
 
-All services, applications, and consumers MUST have a [CloudWatch Alarm](http://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/AlarmThatSendsEmail.html) set up,
+All services, applications, and consumers MNAY have a [CloudWatch Alarm](http://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/AlarmThatSendsEmail.html) set up,
```

Comment:
Typo, also could this be `SHOULD`? I feel like we should encourage people to set up monitoring but agree that we shouldn't require it.